src/cascadia/TerminalApp/App.idl

```diff
@@ -29,6 +29,8 @@ namespace TerminalApp
 
         String GetTitle();
 
+        FLOAT GetNonClientAreaDragBarWidth();

```

Comment:
curious: Is `FLOAT` or `Float` preferred for idl?

Reply:
Or does it not matter?

Reply:
I don{t think it matters, but I prefer Float better than FLOAT. I will change it.